### PR TITLE
Move NodeObject partly to Database

### PR DESCRIPTION
**Pull request by rsalevsky**
Not finished!

The goal is that crowbar isn't accessing any data from chef and only syncs data to chef is needed.

### CODE REVIEW

File: crowbar_framework/app/models/crowbar_service.rb [hunk 1]
```diff
@@ -423,7 +422,7 @@ def initiate_nodes_upgrade
     pxecfg_subdir = "bios/pxelinux.cfg"
 
     upgrade_nodes.each do |node|
-      boot_ip_hex = node["crowbar"]["boot_ip_hex"]
+      boot_ip_hex = node.crowbar["crowbar"]["boot_ip_hex"]
```
Comment (vuntz) [inline crowbar_framework/app/models/crowbar_service.rb]: Are you sure about that one?

Comment (vuntz) [inline crowbar_framework/app/models/crowbar_service.rb]: Ah, I see it elsewhere too, so I guess it's intended.

**Comment by vuntz** [comment]
@rsalevsky I'd suggest to create a separate PR that does the `Node.find_node_by_name` -> ` Node.find_by_name` change as well as the `node[:crowbar_call]` -> `node.crowbar_wall` change. We should be able to merge that fast, and that would make this PR smaller/easier to review/less prone to conflicts.

**Comment by rsalevsky** [comment]
We don't do this anymore.

**Pull request closed without merging**